File: PYTHON/ejercicio57.py
clave = input("Contraseña: ")

tiene_numero = False

for c in clave:
    if c.isdigit():
        tiene_numero = True

if len(clave) >= 8 and tiene_numero:
    print("Contraseña válida")
else:
    print("Contraseña inválida")

File: JS/ejercicio57.js
let contraseña = "clave123";

let tieneNumero = false;

for (let i = 0; i < contraseña.length; i++) {
    if (!isNaN(contraseña[i]) && contraseña[i] !== " ") {
        tieneNumero = true;
    }
}

if (contraseña.length >= 8 && tieneNumero) {
    console.log("Contraseña válida");
} else {
    console.log("Contraseña inválida");
}